SPEC/FILES.md

```diff
@@ -207,6 +219,8 @@ Returns a Pull Stream, where objects can be written of the forms
 {
   path: '/tmp/myfile.txt', // The file path
   content: <data> // A Buffer, Readable Stream, Pull Stream or File with the contents of the file
+  mode: '0755' // optional string or integer mode to store the entry with. strings will be interpreted as a base 8 number
+  `mtime` Either a ` Date` object, an object with `{ sec, nsecs }` properties or the output of `process.hrtime()`  (default: now)
```

Comment:
`mtime:`?

Also, isn't this also optional?

Reply:
Fixed!